src/views/Simulation/utils/pixiUtils.js

```diff
@@ -411,9 +411,22 @@ const generateTextures = (app) => {
   return textures;
 };
 
+export const freeDisplayMemory = (container) => {
+  container.children.forEach((child) => {
+    child.destroy({
+      children: true,
+      texture: true,
+      context: true,
+      style: true,
+    });
+  });
+};
+
```

Comment:
The function name `freeDisplayMemory` does not clearly indicate what the functions does. Can we rename it to `destroyContainerChildren`?